functions/src/index.ts

```diff
@@ -63,6 +63,10 @@ export const scheduledSyncNotionEventsToFirestore = functions.pubsub.schedule('0
 			}
 
 			const title = page.properties["Name"]?.title[0]?.plain_text ?? "Untitled Event";
```

Comment:
```suggestion
			const title = page.properties["Name"]?.title[0]?.plain_text ?? "";
```

Reply:
Already fixed! I caught that the SECOND after I pushed the first commit, fixed it right after in https://github.com/LaurierCS/Website/pull/138/commits/872f044737799bee8af428e38b68145b0f9407de 😭